lmfit/parameter.py

```diff
@@ -5,6 +5,8 @@
 
 from asteval import Interpreter, get_ast_names, valid_symbol_name
 from numpy import arcsin, array, cos, inf, isclose, sin, sqrt
+from numpy import version as npvers
+from packaging.version import Version
```

Comment:
I would rather not add a dependency for such a simple version check. In a few places we just compare the version for SciPy just using: ```int(scipy_version.split('.')[1]) >= 11:`` any reason we can't do that here, especially since it's a short-term need anyway?

Reply:
@reneeotten Yes, I came to the same conclusion, and was just making a similar change too (`npversion.version.startswith('2')`)